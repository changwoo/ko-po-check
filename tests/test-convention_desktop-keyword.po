# -*-text-*-
# freedesktop.org desktop Keyword 번역 검사
#
msgid ""
msgstr ""
"Project-Id-Version: test 1\n"
"POT-Creation-Date: 2000-01-01 00:00+0900\n"
"PO-Revision-Date: 2000-01-01 00:00+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

# 문제되는 경우.
#: ../data/test.desktop.in.in.h:1
msgid "Keyword;"
msgstr "키워드;"

# 올바른 키워드 번역.
#: ../data/test.desktop.in.in.h:1
msgid "Keyword;"
msgstr "Keyword;키워드;"

# 키워드 아님.
#: ../data/test.desktop.in.in.h:1
msgid "Keyword; it's a keyword"
msgstr "키워드입니다"

# 키워드 아님.
#: ../data/test.c:1
msgid "Hmm it's not a keyword;"
msgstr "키워드 아님;;"

# 세미콜론 빠뜨린 경우
#: ../data/test.desktop.in.in.h:1
msgid "Keyword1;Keyword2;"
msgstr "키워드1;키워드2"

# 번역한답시고 약자라고 번역 안 하고 키워드 남겨둔 경우
#: ../data/gnome-tweak-tool.desktop.in.h:3
msgid "Keyword;XKB;"
msgstr "Keyword;키워드;XKB;XKB;"
