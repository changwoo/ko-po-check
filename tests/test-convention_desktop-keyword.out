-:18: Keyword: 검색어 번역에는 원문 키워드를 포함해야 합니다
-:38: 검색어 번역은 ';'으로 끝나야 합니다
-:43: XKB: 검색어가 중복되어 있습니다
